tools/featureGeneration/lcstats.py

```diff
@@ -16,9 +16,252 @@
 from scipy.optimize import curve_fit
 from scipy.signal import sawtooth
 
+from numba import jit
+
 warnings.filterwarnings("ignore")
 
 
+# Define bin intervals for dmdt histograms
```

Comment:
Is this better in the config? Or do you prefer it in the code?

Reply:
Seems better to put it in the config now that you mention it.